ci-operator/config/openshift/release/openshift-release-master.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.12
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: core-valid
```

Comment:
So we previously did not have ci-op driven tests as it makes for a nice circular dependency when ci-op repo goes down and something is goofed. These tests wouldn't be that hard to write in `prow` YAML, could we do that?

```yaml
  - agent: kubernetes
    always_run: true
    branches:
    - master
    context: ci/prow/core-dry
    decorate: true
    name: pull-ci-openshift-release-master-core-dry
    rerun_command: /test core-dry
    spec:
      containers:
      - args:
        - check
        command:
        - make
        image: golang:1.12
        imagePullPolicy: Always
    trigger: (?m)^/test( | .* )core-dry,?($|\s.*)
```

Reply:
Well, technically we could... I really like ci-op jobs b/c they are easier to write, we get rehearsals and all that. I hear what you are saying, but don't think this job is that critical. The situation when we lose `ci-operator` is probably serious enough to warrant an `/override` or button-merge? I'll change it if you insist, but I'd rather keep them ci-op based.

Reply:
True, with `/override` we're OK i think.